lib/tasks/applications.rake

```diff
@@ -21,10 +21,12 @@ namespace :applications do
     puts "config.oauth_secret = '#{a.secret}'"
   end
 
-  desc 'Updates domain name for applications'
-  task :migrate_domain => :environment do
+  desc 'Updates domain name for applications, optionally pass in apps to ignore'
+  task :migrate_domain, [:apps_to_ignore] => :environment do |t, args|
+    apps_to_ignore = args.any? ? args[:apps_to_ignore].split(' ') : []
     raise "Requires OLD_DOMAIN + NEW_DOMAIN specified in environment" unless ENV['OLD_DOMAIN'] && ENV['NEW_DOMAIN']
     Doorkeeper::Application.find_each do |application|
+      next if apps_to_ignore.include?(application)
```

Comment:
good point, it would have to be application[:name]. I will change to use `where.not`. Thanks @bradleywright